overlord/ifacestate/ifacemgr_test.go

```diff
@@ -451,7 +451,7 @@ func (s *interfaceManagerSuite) TestDoSetupProfilesAddsImplicitSlots(c *C) {
 	// Ensure that we have slots on the OS snap.
 	repo := mgr.Repository()
 	slots := repo.Slots(snapInfo.Name())
-	c.Assert(slots, HasLen, 17)
+	c.Assert(slots, HasLen, 18)
```

Comment:
I will remove the requirement to patch this code so we may end up removing it from this branch.